_posts/2024-09-01-devtools.md

```diff
@@ -0,0 +1,125 @@
+---
+layout:     post
+tags:       blog
+title:      "Devtools deep dive: inspect HTML, JavaScript console, and more!"
+date:       2024-09-01
+summary:    "Overview of the fixes to the devtools support during the Outreachy internship"
+categories:
+---
+
+<figure class="_figr"><a href="{{ '/img/blog/devtools-inspector.png' | url }}"><img src="{{ '/img/blog/devtools-inspector.png' | url }}"
+    alt="On the left, it shows the DOM inspector with the tree view, CSS list and computed properties views. On the right is servoshell with servo.org opened."></a>
+<figcaption>The HTML and CSS inspector is able to display the DOM elements and their attributes and CSS properties.</figcaption></figure>
+
+<span class=_floatmin></span>Servo has been working on improving our [Firefox devtools](https://firefox-source-docs.mozilla.org/devtools-user) support as part of the [Outreachy](https://www.outreachy.org) internship program since June, and we're thrilled to share significant progress.
+
+Devtools are a set of **web developer tools inside Firefox** that allows you to examine, edit, and debug HTML, CSS, and JavaScript.
+Servo leverages existing work there and uses the same open protocol to inspect its own websites.
```

Comment:
this reads a little odd because devtools like this exist in all of the major browsers. i wonder if we can reword it to move the Firefox mention to the second sentence, so we go from general (devtools are browser web developer tools) to specific (we use the Firefox devtools)?

Reply:
it may also be worth adding something about how Firefox lets you point their devtools to another Firefox, like on a mobile device, which is why they have a protocol we can use too.

Reply:
Thank you for all of the changes, it reads great now. How about something like this for this paragraph?

Reply:
looks good!

Reply:
looks good, thanks!